src/index.js

```diff
@@ -336,7 +336,8 @@ function removeNodeFromHost(host, node) {
 }
 
 function removeSlotFromRoot(root, node) {
-  node.assignedNodes().forEach(slotNodeFromSlot);
+  const assignedNodes = Array.prototype.slice.call(node.assignedNodes());
```

Comment:
Yeah I know it gets transpiled, but IIRC it transpiles to `Array.from()` which is also ES2015 and needs a polyfill.